### PR TITLE
[DOCS-10133] [DOCS-10131] Review/edit CloudBees CD/RO blueprint add-on documentation and diagrams

### PR DESCRIPTION
**Pull request by kellie-freeman**
First pass edits for the CloudBees CD/RO blueprint add-on documentation and diagrams.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,23 +1,57 @@
...
-This document provides guidelines for contributing to the module.
+This document provides guidelines for contributing to the CloudBees CD/RO add-on for Amazon EKS blueprints.
 
 ## Dependencies
 
-Validate your changes inside the blueprint-agent described in [.Dockerfile](blueprints/Dockerfile). It can be run `make dBuildAndRun`.
+Validate your changes inside the blueprint agent, as described in [Dockerfile](blueprints/Dockerfile) For example, it can be used to run `make dBuildAndRun`.
 
-## Pre-commits: Linting, Formatting and Secrets Scanning
+<!---
```
Comment (kellie-freeman) [inline CONTRIBUTING.md:9]: I added the following sections, but commented them out for now, so we can simply remove the comments once we're ready to use them.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# CloudBees CD Add-on for AWS EKS
+# CloudBees CD/RO add-on for Amazon EKS blueprints
 
 <p align="center">
   <a href="https://www.cloudbees.com/capabilities/continuous-delivery"><img alt="cloudbees-icon" src="https://images.ctfassets.net/vtn4rfaw6n2j/7FKeUjwsXI1d2JPUIvSMZJ/be286872ace9ca3b6b66a64adbb3c16a/cb-tag-sm.svg?fm=webp&q=85" height="120px" /></a>
```
Comment (kellie-freeman) [inline README.md:4]: Do we have another hosted image we can use with the new branding, to replace this one?

**Review by ps-ssingh — APPROVED** [review]

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,3 +1,11 @@
 * @carlosrodlop
 * @holywen
-* @ps-ssingh
\ No newline at end of file
+* @ps-ssingh
+
+# Center of Excellence team as owners for files that require documentation review.
+
+*.md @cloudbees/team-docs-coe
```
Comment (kellie-freeman) [inline .github/CODEOWNERS:7]: @carlosrodlop - do you mind updating, so `team-docs-coe` has write access to the repo?

Comment (kellie-freeman) [reply .github/CODEOWNERS]: I will open a separate PR for this later.

**Review by holywen — APPROVED** [review]

**Comment by holywen** [comment]
Thanks Kellie!

File: .github/CODEOWNERS (hunk 3 above)
Comment (carlosrodlop) [reply .github/CODEOWNERS]: I will do it @kellie-freeman !

**Review by carlosrodlop — APPROVED** [review]

**Pull request merged**